### ISSUE TITLE
Org-Noter and seperate note files

### ISSUE THREAD

**Issue by petemcd18** [weirdNox/org-noter#67]
Hi,
I may be asking a daft question but I went to try out this package following previously using Interleaving Mode. 

My setup
Path where I keep all PDFs ~/Documents/pdfs/
Path where I keep all notes as individual note files ~/Documents/notes/

citekey = authoryear
authoryear.pdf
authoryear.org

I appear to be having problems getting org-noter to understand that I have a seperate note file for each pdf. It seems to want to have one .org file for all notes.?

I've set (custom-set-variables '(org-noter-notes-search-path (quote ("/Documents/notes/"))))

I've also added 
(setq org-noter-property-doc-file "INTERLEAVE_PDF"
      org-noter-property-note-location "INTERLEAVE_PAGE_NOTE")
(setq org-noter-notes-window-location 'vertical-split) into my init.el

which I hoped would enable me simply to open a pdf, do M-x org-noter and then bang my previous interleave .org note file would open for the corresponding pdf I am viewing but this is not the experience I am having.

I am not a coder, so I'm probably doing something incorrectly. I would be grateful for any advice here. Thanks

**Comment by weirdNox** [comment]
Hi there!
I noticed you don't have a tilde in `(custom-set-variables '(org-noter-notes-search-path (quote ("/Documents/notes/"))))`. Could that be your problem? I believe you want `(custom-set-variables '(org-noter-notes-search-path (quote ("~/Documents/notes/"))))`. Also, make sure the folder `~/Documents/notes/` exists before running the command.

I don't use this feature very much, but I just tested it and seems to be working.

It will search all parent directories and search paths for files annotating the document. If it doesn't find one, it should either prompt you for the notes file name and location or choose one if it already exists in a search path.
